src/extension/code-actions/MissingImportCreator.ts

```diff
@@ -7,6 +7,8 @@ import { DeclarationIndexMapper } from '../utilities/DeclarationIndexMapper';
 import { AddImportCodeAction, AddMissingImportsCodeAction, NoopCodeAction } from './CodeAction';
 import { CodeActionCreator } from './CodeActionCreator';
 
+const REGEX_CANNOT_FIND_IMPORT = /cannot find name (['"])(.*)\1/i;
```

Comment:
- Ensuring quote match the usual way (group backreference)
- Stripping useless costly `g` flag